app/graphql/cache.ts

```diff
@@ -190,5 +204,42 @@ export const createCache = () =>
           },
         },
       },
+      UsdWallet: {
```

Comment:
Line 188 will have a bug if the display currency does not have two decimals. We need to use the decimal places of the display currency. Same for line 239.

Reply:
this `minorUnitToMajorUnitOffset`?

Reply:
Yes

Reply:
have added a TODO for it